utils/swift_build_support/swift_build_support/products/swiftsyntax.py

```diff
@@ -116,13 +116,7 @@ def test(self, host_target):
         llvm_build_dir = os.path.realpath(llvm_build_dir)
 
         self.run_swiftsyntax_build_script(target=host_target,
-                                          command='test',
-                                          additional_params=[
-                                              '--filecheck-exec',
-                                              os.path.join(llvm_build_dir,
-                                                           'bin',
-                                                           'FileCheck')
-                                          ])
+                                          command='test')
 
     def should_install(self, host_target):
         return self.args.install_swiftsyntax
```

Comment:
Are all the dependencies in `get_dependencies` actually needed?

Reply:
Yes, I think so. We might not need to list `LLVM`, `LibCXX`, `LibICU` and `LLBuild` because they are transitive dependencies of Swift / SwiftPM that we don’t explicitly depend upon but I’m not sure if `get_dependencies` should include these transitive dependencies. 

@gottesmm You added these, do you know if we explicitly need to list these dependencies here?